code/upstream-integration-tests/src/androidTest/java/com/adobe/marketing/mobile/edge/integration/UpstreamIntegrationTests.kt

```diff
@@ -337,15 +341,16 @@ class UpstreamIntegrationTests {
                 }
               ]
             }
-            """.trimIndent()
+            """
 
             // Unsafe access used since testSendEvent_receivesExpectedEventHandles guarantees existence
             val locationHintResult = TestSetupHelper.getEdgeEventHandles(expectedHandleType = TestConstants.EventSource.LOCATION_HINT_RESULT)
                 .first()
-            JSONAsserts.assertTypeMatch(
-                expected = JSONObject(expectedLocationHintJSON),
-                actual = JSONObject(locationHintResult.eventData),
-                exactMatchPaths = listOf("payload[*].scope", "payload[*].hint")
+            assertTypeMatch(
```

Comment:
As discussed, we can do assertExactMatch here and typeMatch the "payload[*].ttlSeconds".